docker-vms/vumi_metrics/config/supervisor/vumi_metrics.conf

```diff
@@ -0,0 +1,62 @@
+[supervisord]
+nodaemon=true
+
+[program:graphite_metrics_collector]
+command=twistd -n --pidfile=graphite-metrics-collector-%(process_num).pid \
```

Comment:
We probably don't want `process_num` here, since we're only running one of these?